WordPress/src/main/java/org/wordpress/android/ui/qrcodeauth/compose/components/SecondaryButton.kt

```diff
@@ -0,0 +1,37 @@
+package org.wordpress.android.ui.qrcodeauth.compose.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.ui.compose.unit.Margin
+
+@Composable
+fun SecondaryButton(
+    onClick: () -> Unit,
+    enabled: Boolean = true,
+    text: String
+) {
+    Button(
+            onClick = onClick,
+            enabled = enabled,
+            elevation = ButtonDefaults.elevation(0.dp),
+            colors = ButtonDefaults.buttonColors(
+                    backgroundColor = MaterialTheme.colors.background,
+                    disabledBackgroundColor = MaterialTheme.colors.background,
+            ),
+            modifier = Modifier
+                    .fillMaxWidth()
```

Comment:
`fillMaxWidth` causes the parents to expand too to fill all available space (check this StackOverflow [question](https://stackoverflow.com/questions/65942711/match-width-of-parent-in-column-jetpack-compose) about this), and this might not be what you want in all cases, I'd say remove it from here, and allow consumers to add it when needed by adding the `Modifier` argument as stated above.

Reply:
Solved in [0b9b0b7](https://github.com/wordpress-mobile/WordPress-Android/pull/16934/commits/0b9b0b7c2ed0902e034373d6aca9ce61f07563db)